actors/builtin/miner/monies.go

```diff
@@ -62,6 +62,11 @@ const ConsensusFaultFactor = 5
 var LockedRewardFactorNum = big.NewInt(75)
 var LockedRewardFactorDenom = big.NewInt(100)
 
+// Base reward for successfully disputing a window posts proofs.
+var BaseRewardForDisputedWindowPoSt = big.Lsh(big.NewInt(4), math.Precision128) // PARAM_SPEC
```

Comment:
I assume the intent here is "4 FIL". This doesn't look right, there's no 128-bit fixed point estimation going on here. A form that is more transparently correct and used elsewhere (builtin/network.go, reward_logic.go) is:

> var BaseRewardForDisputedWindowPoSt = big.Mul(big.NewInt(4), builtin.TokenPrecision)

Reply:
:roll_eyes: thanks for that.